iai_donbot_description/robots/iai_donbot_spawn.xacro

```diff
@@ -5,7 +5,7 @@
 
```

Comment:
I would suggest calling this file `iai_donbot.urdf.xacro`.

Reply:
I saw that kind of naming sometimes but still don't get why. Can you explain it to me?

Reply:
I changed it

Reply:
Its to indicate that if you run `xacro iai_donbot.urdf.xacro`, the result will be a proper urdf.

Reply:
I meant that iai_donbot_spawn.xacro gets renamed to iai_donbot.urdf.xacro. and iai_donbot.xacro keeps its name.

Reply:
ok I see